mmpose/core/post_processing/post_transforms.py

```diff
@@ -226,3 +226,55 @@ def rotate_point(pt, angle_rad):
     rotated_pt = [new_x, new_y]
 
     return rotated_pt
+
+
+def fliplr_interference_joints(joints, width, matched_parts):
+    """flip interference joint."""
```

Comment:
1. what is an interference joint?
2. this is an inplace operation. do you need to protect the input by copy?

Reply:
![image](https://user-images.githubusercontent.com/9464825/89768782-aa474f00-db2e-11ea-8d01-0926e152d272.png)

Reply:
a joint in someones bbox but not belong him

Reply:
it will be good to add this explanation to docstring